administration/intro.adoc

```diff
@@ -265,11 +266,47 @@ ____________________________________
 Note: Updates are not yet supported.
 ____________________________________
 
-When using the stable deployment flow, usually it is sufficient to re-apply the manifests for performing a
-rolling update:
+In both cases re-apply the manifests will not guarantee you the correct rolling update:
```

Comment:
doesn't 0.17 now support rolling upgrades?

Reply:
I will wait for @davidvossel response

Reply:
yes, the plan is to support rolling updates from 0.17.0 onward.